### PR TITLE
Update logging

### PR DESCRIPTION
**Pull request by erri120**
- Improved logging messages containing more information
- We're now using two logging files:
  - `nexusmods.app.main` for the main process only
  - `nexusmods.app.slim` for the "slim" process only
- Updates the "bug" issue template to prompt the user to upload log files.

Having two logging files fixes the issue where both the main and slim process try to write to the same file. While we didn't have any corruptions of log files, we did have an issue where a slim process will move the current main log to an archive log while the main process would continue writing to the archive log. It made debugging and viewing logs difficult, this PR fixes that issue by clearly separating the two logs.

### CODE REVIEW

File: src/NexusMods.App/AppConfig.cs [hunk 1]
```diff
@@ -64,15 +64,13 @@ public void Sanitize(IFileSystem fs)
...
 
-    /// <summary>
-    /// Path to historical log files, with templated element e.g. 'PATH_TO_FILE/nexusmods.app.{##}.log'
-    /// </summary>
-    public ConfigurationPath ArchiveFilePath { get; }
+    public ConfigurationPath MainProcessArchiveFilePath { get; }
+
+    public ConfigurationPath SlimProcessLogFilePath { get; }
+
+    public ConfigurationPath SlimProcessArchiveFilePath { get; }
```
Comment (Al12rs) [inline src/NexusMods.App/AppConfig.cs:73]: Missing docs

Comment (erri120) [reply src/NexusMods.App/AppConfig.cs:73]: Do you really need/want docs for this?

Comment (Al12rs) [reply src/NexusMods.App/AppConfig.cs:73]: `ArchiveFilePath` might not be immediate, some explanation for what is SlimProcess might be useful as well

**Review by Al12rs — APPROVED** [review]
Looks ok if there are no objections to splitting the log file from others

**Review by halgari — COMMENTED** [review]
Looks good to me, good solution to the problem

**Pull request merged**